src/components/tasks/card/progressBar/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC, useState } from 'react';
+import { useRouter } from 'next/router';
+
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import {
+    useUpdateSelfTaskMutation,
+    useUpdateTaskMutation,
+} from '@/app/services/tasksApi';
+import useUserData from '@/hooks/useUserData';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import HandleProgressText from './ProgressText';
+import HandleProgressbar from './ProgressBar';
+import { ERROR_MESSAGE, PROGRESS_SUCCESSFUL } from '@/constants/constants';
+
+import classNames from '@/components/tasks/card/card.module.scss';
+import { ProgressContainerProps } from '@/interfaces/task.type';
```

Comment:
These newlines between import statements aren't required. I understand you want to group similar kinds of imports here, but that's not required. It is a general practice to keep it all together.

Reply:
fixed